### ISSUE TITLE
CDN (like unpkg.com)

### ISSUE THREAD

**Issue by mik-jozef** [dhershman1/vue-debounce#62]
**Version of Vue**
3.2.22

**Browser**
Chrome 96

Can I use vue-debounce with a cdn, like unpkg.com? I have a simple website whose source are static files, and don't even use npm.

it would be great if I could just do something like:

```ts
import vueDebounce from 'https://unpkg.com/vue-debounce@3.0.1/dist/debounce.min.js';

// or

import vueDebounce from 'https://unpkg.com/vue-debounce@3.0.1/src/index.js';
```

However the former does not work because it is not an ES6 module, and the later fails with a cryptic error:

```error
Uncaught TypeError: debouncedFn is not a function
    at directive.js:84
```

If this is somehow possible, I'd appreciate a mention in the docs, if not, then this is a feature request :D

**Comment by dhershman1** [comment]
Hmm, this is a good point. It was working to an extent, but the shopped distro isn't module compatible sadly.

Probably just need to have a 3rd rollup build that makes the code module friendly and point unpkg to one of those distros.

**Comment by dhershman1** [comment]
I just released v3.0.2, if you still want to try this please do. I was able to get it working on my testing let me know if you have any issues.
